cv32e40x/tests/cfg/default.yaml

```diff
@@ -4,8 +4,7 @@ compile_flags:
     +define+NO_PULP
 ovpsim: >
     --override root/cpu/misa_Extensions=0x001104
-    --override root/cpu/marchid=4
-    --override root/cpu/noinhibit_mask=0xFFFFFFF0
+    --override root/cpu/marchid=4    
```

Comment:
Perhaps an idea to harmonize this parameter with the format used by gcc/binutils? i.e. something like "rv32imc" (I guess we could create a define/enum to parameterize this)

Reply:
These are defined by Imperas and thus are part of their OVPSim IP.
Ideally I would like to completely fold these into the automatic control file generation so the logic is consolidated for generating OVPSim control file.